package.py

```diff
@@ -0,0 +1,911 @@
+            logger.error(f'{destination} was not a fat binary, only has {archs}')
+            exit(1)
+
+    @staticmethod
+    def modify_wheel_metadata(wheel_metadata: Path) -> None:
+        with open(wheel_metadata, 'r') as file:
+            data = file.readlines()
+            for (index, line) in enumerate(data):
+                if not line.startswith('Tag'):
+                    continue
+                data[index] = line.replace('x86_64', 'universal2')
+
+        with open(wheel_metadata, 'w') as file:
+            file.writelines(data)
+
+    def __download_patched_pip(self) -> Path:
+        pip_wheel = 'pip-22.1.2-py3-none-any.whl'
+        response = requests.get(f'https://github.com/rotki/rotki-build/raw/main/{pip_wheel}')
+        if response.status_code == 200:
+            temporary_directory = self.__storage.temporary_directory
+            temporary_directory.mkdir(exist_ok=True)
+            wheel_file = temporary_directory.joinpath(pip_wheel)
+            with open(wheel_file, 'wb') as file:
+                file.write(response.content)
+            return wheel_file
+
+        logger.error(f'pip failed to download {pip_wheel}')
+        raise Exception(f'{pip_wheel} download failed')
+
+    def __get_versions(self, packages: list[str]) -> dict[str, str]:
+        requirements = self.__storage.working_directory.joinpath('requirements.txt')
+        package_versions: dict[str, str] = {}
+        with open(requirements) as fp:
+            while True:
+                line = fp.readline()
+
+                if not line:
+                    break
+
+                if not line.strip() or line.startswith('#'):
+                    continue
+                requirement = line.split('#')[0]
+                req = requirement.split(';')
+                requirement = req[0]
+                if len(req) > 1:
+                    if req[1].strip() == "sys_platform == 'win32'":
+                        continue
+
+                s = requirement.split('==')
+                if s[0] in packages:
+                    package_versions[s[0].strip()] = s[1].strip()
+        return package_versions
+
+    @log_group('coincurve universal2 wheel')
+    def __coincurve_universal_repackage(self) -> None:
+        storage = self.__storage
+        logger.info('Preparing to merge coincurve wheels')
+        package_name = 'coincurve'
+        versions = self.__get_versions(packages=[package_name])
+        build_directory = storage.build_directory
+        if not build_directory.exists():
+            build_directory.mkdir(parents=True)
+
+        temp = build_directory.joinpath('temp')
+        temp.mkdir(parents=True, exist_ok=True)
+
+        wheels_directory = build_directory.joinpath("wheels")
+        wheels_directory.mkdir(parents=True, exist_ok=True)
+
+        x86_64 = temp.joinpath('x86_64')
+        arm64 = temp.joinpath('arm64')
+
+        package = f'coincurve=={versions.get("coincurve")}'
```

Comment:
You defined the `package_name` constant in this function but is used only once.
```suggestion
        package = f'{package_name}=={versions.get(package_name)}'
```